usmqe_tests/api/gluster/conftest.py

```diff
@@ -15,6 +15,26 @@ def valid_cluster_id(valid_session_credentials):
     return cluster_list[0]["cluster_id"]
 
 
+@pytest.fixture
+def valid_nodes(valid_session_credentials):
```

Comment:
This returns active nodes from which you can create cluster. There shouldn't be any gluster installed.